scripts/release/README.md

```diff
@@ -0,0 +1,31 @@
+# Release Scripts
```

Comment:
Can you add some details to add detail about the release process in `CONTRIBUTING.md` and add a reference to this file for further details about the release command?

The `CONTRIBUTING.md` should cover:
- How to release a stable version
- How to release an alpha version

Reply:
I'm not so sure `CONTRIBUTING.md` is a good place for the release process. Something like a `docs/release.md` might be a better fit, but a shared document might be best for now since it sounds like `nucleus` is supposed to handle package deployment in the near future.

Reply:
Sounds good to me.